src/users/users.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { UsersRepository } from './users.repository';
+import { Prisma, User } from '@prisma/client';
+import { userData } from 'src/auth/interfaces/user-data.interface';
+
+@Injectable()
+export class UsersService {
+  constructor(private readonly usersRepository: UsersRepository) {}
+
+  // 검증용
+  async checkUserExists(where: Prisma.UserWhereUniqueInput): Promise<boolean> {
+    return this.usersRepository.exists(where);
+  }
+
+  // 비밀번호 확인용 유저 조회
+  async findUserForAuth(username: string): Promise<User | null> {
+    return this.usersRepository.findOneByUsername(username);
+  }
```

Comment:
구현을 보면 username에 해당하는 유저를 조회한다고 보이는데 findUserByUsername같은 메서드명은 어떤가요??

```suggestion
  async findUserForAuth(username: string): Promise<User | null> {
    return this.usersRepository.findOneByUsername(username);
  }
```

Reply:
username으로 유저를 찾는 메서드라서 더 직관적인 것 같아요. 감사합니다!